addons/road-generator/ui/road_toolbar_create_menu.gd

```diff
@@ -29,40 +24,57 @@ enum CreateMenu {
 	LANE,
 	LANEAGENT,
 	TWO_X_TWO,
-	EXPORT_MESH
+	EXPORT_MESH,
+	LOCK_ROTATION_X,
+	LOCK_ROTATION_Y = 90,  # for some reason, this being `9` breaks the checkbox
+	LOCK_ROTATION_Z,
 }
 
 enum MenuMode {
 	STANDARD,
-	SAVED_SUBSCENE, # Don't offer to create children
-	EDGE_SELECTED, # Not yet used, could offer intersections/next pieces to add.
+	SAVED_SUBSCENE,  # Don't offer to create children
+	EDGE_SELECTED,  # Not yet used, could offer intersections/next pieces to add.
 }
 
+const ICN_CT = preload("../resources/road_container.png")
+const ICN_RP = preload("../resources/road_point.png")
+const ICN_LN = preload("../resources/road_lane.png")
+const ICN_AG = preload("../resources/road_lane_agent.png")
+const RcSubMenu = preload("./rc_submenu.gd")
```

Comment:
Moved these down to match the [suggested code order](https://docs.godotengine.org/en/4.4/tutorials/scripting/gdscript/gdscript_styleguide.html#code-order)